content/company/strategy/code-graph/search/index.md

```diff
@@ -11,21 +11,7 @@ Quicklinks:
 
 ## Mission
 
-Sourcegraph's mission is to make it so that everyone can code. The Search teams' mission is to enable developers to find the code they need when they need it.
-
-## Vision
-
-#### 1 Year vision
-
-> Enterprise customers of Sourcegraph actively use our differentiated code search features. Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results.
-
-#### 3 Year vision
-
-> Search powers [unified](../index.md#deliver-a-unified-experience) use cases of Sourcegraph, at any Enterprise scale and across the entire OSS universe.
-
-#### 10 year vision
-
-> Code search is a cornerstone of [universal coding literacy](../../index.md#ten-year-vision-democratize-coding).
+Sourcegraph's mission is to make it so that everyone can code. The Search teams' mission is to enable developers to find the code they need when they need it. We want enterprise customers of Sourcegraph actively use our differentiated code search features. Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results. Eventually, we envision that Search will power [unified](../index.md#deliver-a-unified-experience) use cases of Sourcegraph, at any Enterprise scale and across the entire OSS universe. Beyond that, we see Code search is a cornerstone of [universal coding literacy](../../index.md#ten-year-vision-democratize-coding).
```

Comment:
Maybe the company mission doesn't need to be restated here since we aren't tying it to the team mission.

Reply:
```suggestion
The Search team's mission is to enable developers to find the code they need when they need it.

In the near term, we want to make sure our customers are aware of and using our differentiated code search features. For example, Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results. We envision that Search will power [unified](../index.md#deliver-a-unified-experience) use cases of Sourcegraph, at any scale and across the entire OSS universe.

Reply:
@benvenker @lguychard since this wasn't moving forward I made a small edit to just have the sentences flow a little better and will merge.

I removed the part about how search is a cornerstone of universal coding literacy, because we didn't say how and I wasn't really sure how to say that myself. Please do feel free to re-add (I like the inspirational element) but when you do include why.